### ISSUE TITLE
[Suggestion]  Small, Modular, or Resizable Text Display Screen & [Question] Special Character Input

### ISSUE THREAD

**Issue by WILDCATreactor** [tsunamayo/Starship-EVO#3085]
Suggestion: Eventual inclusion of a 0.25mx1m Display Screen would be nice for compact signage, particularly nameplates for rooms and smaller direction signposts.  It's definitely another block that offers good candidacy for being made resizable.  I also assume the welding tool, once implemented, would work on display screens to allow us to make really massive signage? or would you just allow for 16mx16m scaling?

Question: how do I select or input the special characters? I'd tried various hotkeys like [crtl] and [alt] like you would for ASCI inputs, but no luck.

**Comment by ProPeach** [comment]
To answer the question, use <sprite=000> where 000 is the code displayed for the sprite on the right of the screen

**Comment by tsunamayo** [comment]
Yes I will do a refactoring of text screen and so on to allow more customization options. But letter decals are a must first I think.
Btw weld tool wont allow this...
Thanks

**Comment by WILDCATreactor** [comment]
alphanumeric decals would be awesome too, for some reason I was thinking we'd have to use the box/line and curve stickers to make all of those at a large scale
